fades/parsing.py

```diff
@@ -232,12 +233,29 @@ def parse_manual(dependencies):
     return _parse_requirement(dependencies)
 
 
+def _read_lines(filepath):
+    """Read a req file to a list to support nested requirement files."""
+    lines = []
+    with open(filepath, 'rt', encoding='utf8') as fh:
+        for line in fh:
+            line = line.strip()
+            if line.startswith("-r"):
+                logger.debug(
+                    "Reading deps from nested requirement file: %s", line)
+                nested_filename = line.split()[1]
```

Comment:
if the reqs file has only a "-r" line, this will crash (yes! it's an error in the req file, but we should not crash! maybe log in warning and ignore the line?)

Reply:
updated!